docs/source/installation.rst

```diff
@@ -95,7 +93,6 @@ Windows
 3. Install GPkit
 ++++++++++++++++
   - Run ``pip install gpkit`` at an Anaconda Command Prompt.
-  - Run ``pip install pint`` for units support (recommended)

```

Comment:
hmm, my fault for not thinking this through carefully. @galbramc, yes, your plan to print the pint version number sounds excellent.

Note the dependency change won't happen until this is merged.